test/verify/check-networkmanager-wireguard

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/../common/pywrap", sys.argv)
+
+import subprocess
+import sys
+
+import netlib
+import packagelib
+import testlib
+
+
+class TestWireGuard(packagelib.PackageCase, netlib.NetworkCase):
+    provision = {
+        "machine1": {"address": "192.168.100.11/24", "restrict": False},
+        "machine2": {"address": "192.168.100.12/24", "restrict": False},
```

Comment:
Reduce memory of at least machine2. Also, this ought to work without `restrict`, I suppose that was a testing leftover.

Reply:
done